### ISSUE TITLE
annoying red color when the background is transparent

### ISSUE THREAD

**Issue by hakan-demirli** [QNetITQ/WaveFox#186]
I am not sure if this is a firefox bug, wayland bug or a hyprland bug. I am on linux and using Hyprland tiling manager.

The transparent parts are getting redder and redder. Only firefox is doing that so I thought this is some CSS rule from firefox messing up the background. The color resets If I change the focus.

Firefox version: `120.0.1 (64-bit)`
Distro: Arch Linux
WM: Hyprland-git

**Comment by hakan-demirli** [comment]
Ok. I manage to reproduce the issue with this much CSS.
```css
#main-window,
#tabbrowser-tabpanels {
  background: transparent !important;
}
:root {
  background-color: transparent !important;
}
```
Here is a video demonstration of the bug.

https://github.com/QNetITQ/WaveFox/assets/78746991/b957160c-1716-4750-8464-5621375439ac




I don't really know why there is a growing red plane whenever tabs are reloading or the focus of the window is regained.

**Comment by QNetITQ** [comment]
Check the key `gfx.webrender.all`

**Comment by hakan-demirli** [comment]
I did check it and restarted firefox couple of times. I also created a new profile which also did not work. I switched my GTK theme from Adawatia to Dracula to Nord to no avail.

> Check the key `gfx.webrender.all`

![image](https://github.com/QNetITQ/WaveFox/assets/78746991/2dc0de96-a421-4436-b279-c3a5c90fdeb1)

https://github.com/QNetITQ/WaveFox/blob/127ef83ed69fe864e56885a1756bb493798f09c9/README.md?plain=1#L118

I can turn the whole window transparent using this window rule in Hyprland.
```
windowrule=opacity 0.95 0.95,(firefox)$
```
I will use this as a workaround until I identify the root cause of the issue. Also, I just noticed that the problem might be solved via CSS but it is definitely not a WaveFox specific problem. So, apologies for this unnecessary issue. I will ask the guys at r/FirefoxCSS. Thank you.
